### PR TITLE
Add `tenants` RBAC for Console

### PR DESCRIPTION
**Pull request by brionmario**
### Proposed changes in this pull request

- Add RBAC config for `Multi Tenancy` feature in the new React Console

### Related Issues

- https://github.com/wso2/product-is/issues/21178

### Related PRs

- https://github.com/wso2/identity-apps/pull/6962

### CODE REVIEW

File: features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/org.wso2.carbon.identity.core.server.feature.default.json [hunk 1]
```diff
@@ -1377,6 +1377,17 @@
...
   "console.server.scopes.update": ["internal_governance_update"],
   "console.server.scopes.delete": [],
+  "console.tenants.enabled": true,
+  "console.tenants.disabled_features": [
+    "tenants.add.tenant.from.dropdown",
+    "tenants.make.default",
+    "tenants.organizations.quick.nav.from.dropdown"
+  ],
+  "console.tenants.scopes.feature": ["console:tenants"],
+  "console.tenants.scopes.create": ["internal_notification_senders_create"],
```
Comment (pavinduLakshan) [inline features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/org.wso2.carbon.identity.core.server.feature.default.json:1387]: Don't we need to have unique scopes for tenant mgt CRUD?

**Review by pavinduLakshan — DISMISSED** [review]

File: features/api-resource-mgt/org.wso2.carbon.identity.api.resource.mgt.server.feature/resources/api-resource-collection.xml [hunk 2]
```diff
@@ -524,6 +524,25 @@
...
+                <Scope name="console:tenants"/>
+            </Feature>
+            <Update>
+                <Scope name="internal_modify_tenants"/>
+            </Update>
+            <Delete>
+                <Scope name="internal_modify_tenants"/>
+            </Delete>
+            <Create>
+                <Scope name="internal_list_tenants"/>
```
Comment (brionmario) [inline features/api-resource-mgt/org.wso2.carbon.identity.api.resource.mgt.server.feature/resources/api-resource-collection.xml:539]: `internal_list_tenants` is what we have in the identity.xml.
Hope this is intended @hwupathum 

https://github.com/wso2/carbon-identity-framework/blob/22b18dfcd8e7f94847bdf4bdc6ecef56bd7119ba/features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2#L3230-L3233

File: features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/org.wso2.carbon.identity.core.server.feature.default.json (hunk 1 above)
Comment (brionmario) [reply features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/org.wso2.carbon.identity.core.server.feature.default.json]: Made a mistake. Fixed now.

File: features/api-resource-mgt/org.wso2.carbon.identity.api.resource.mgt.server.feature/resources/api-resource-collection.xml (hunk 2 above)
Comment (hwupathum) [reply features/api-resource-mgt/org.wso2.carbon.identity.api.resource.mgt.server.feature/resources/api-resource-collection.xml:539]: Yes. This is the existing permission for all the GET methods in tenant management API

Comment (PasinduYeshan) [reply features/api-resource-mgt/org.wso2.carbon.identity.api.resource.mgt.server.feature/resources/api-resource-collection.xml:539]: There's `internal_create_tenants` scope but seems like its not being used in identity.xml.

https://github.com/wso2/carbon-identity-framework/blob/22b18dfcd8e7f94847bdf4bdc6ecef56bd7119ba/features/api-resource-mgt/org.wso2.carbon.identity.api.resource.mgt.server.feature/resources/system-api-resource.xml#L347C13-L348C67

**Review by PasinduYeshan — DISMISSED** [review]

**Review by pavinduLakshan — DISMISSED** [review]

File: features/api-resource-mgt/org.wso2.carbon.identity.api.resource.mgt.server.feature/resources/api-resource-collection.xml (hunk 2 above)
Comment (hwupathum) [reply features/api-resource-mgt/org.wso2.carbon.identity.api.resource.mgt.server.feature/resources/api-resource-collection.xml:539]: Yeah we should use `internal_create_tenants` scope

**Review by hwupathum — CHANGES_REQUESTED** [review]

File: features/api-resource-mgt/org.wso2.carbon.identity.api.resource.mgt.server.feature/resources/api-resource-collection.xml (hunk 2 above)
Comment (brionmario) [reply features/api-resource-mgt/org.wso2.carbon.identity.api.resource.mgt.server.feature/resources/api-resource-collection.xml]: Updated with https://github.com/wso2/carbon-identity-framework/pull/6047/commits/8a50a36f535996eb91db9c18642ae1231f07a080

We should update the `identity.xml` as well right @hwupathum ?

**Review by PasinduYeshan — APPROVED** [review]

File: features/api-resource-mgt/org.wso2.carbon.identity.api.resource.mgt.server.feature/resources/api-resource-collection.xml (hunk 2 above)
Comment (hwupathum) [reply features/api-resource-mgt/org.wso2.carbon.identity.api.resource.mgt.server.feature/resources/api-resource-collection.xml]: Yes, to be used with legacy runtime.

**Review by hwupathum — APPROVED** [review]

**Pull request merged**